.github/workflows/ci-future.yml

```diff
@@ -0,0 +1,47 @@
+name: CI-future
```

Comment:
Out of curiosity, what's the advantage of using a separate workflow for nightlies?

Reply:
to keep that sweet sweet green badge even when things go sideways on nighly

Reply:
Ah, good idea. We could do that in DataFrames too, and in the PR I opened against Example.jl.

Reply:
also, less cheekily, to be able to see at a glance if any of the stable releases have failing tests AND whether nightly has failing tests, since those indicate different types of problems (only the first should block merging IMO, but we can't afford to ignore the second so we don't want to use `continue-on-error` and nighly failures green...).

Reply:
FWIW I learned this trick from the MixedModels CI which has been on actions for a while now...